samples/grids/tree-grid/data-summaries-custom/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Sample | Ignite UI | Web Components | infragistics</title>
+    <meta charset="UTF-8" />
+
+    <link rel="shortcut icon" href="https://static.infragistics.com/xplatform/images/browsers/wc.png" >
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Kanit&display=swap" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Titillium Web" />
+    <link rel="stylesheet" href="https://static.infragistics.com/xplatform/css/samples/shared.v8.css" />
+    <link rel="stylesheet" href="/src/index.css" type="text/css" />
+
+  </head>
+
+  <body>
+    <div id="root">
+
+<div class="container sample ig-typography">
+
+    <div class="container fill">
+        <igc-tree-grid
+        auto-generate="false"
+        name="treeGrid"
+        id="treeGrid"
+        primary-key="ID"
+        foreign-key="ParentID">
+            <igc-column
+            field="Name"
+            data-type="string"
+            pinned="true">
```

Comment:
```suggestion
            data-type="string">
```

This column is not pinned in the [Angular sample](https://www.infragistics.com/products/ignite-ui-angular/angular/components/treegrid/summaries#custom-summaries-which-access-all-data) and the [Blazor sample](https://github.com/IgniteUI/igniteui-blazor-examples/pull/711/files#diff-a746ecc88f359b2f6c7e394074700c394178a33909acada58cee0ab2c594c7e4R14-R17).